app/assets/stylesheets/components/_action-link.scss

```diff
@@ -0,0 +1,92 @@
+.app-c-action-link {
+  display: inline-block;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-align: center;
+  align-items: center;
+  box-sizing: border-box;
+  padding-left: 50px;
+  min-height: 45px;
+  max-width: 460px;
+  background-image: image-url("covid-arrow.png"); // PNG fallback for SVG
+  background-image: image-url("covid-arrow.svg");
+  background-repeat: no-repeat;
+  background-position: 0 50%;
+  background-size: 37px auto;
+
+  @include govuk-media-query($until: tablet) {
+    max-width: 410px;
+    margin-bottom: govuk-spacing(8);
+  }
+
+  @include govuk-media-query($from: desktop) {
+    // IE10/11 specific - vertical align in flexbox is ignored
+    // add padding to action link container to improve arrow/text alignment on IE
+    @media screen and (min-width:0\0) {
+      padding-top: govuk-spacing(2);
+    }
+  }
+}
+
+.app-c-action-link__nowrap-text {
+  white-space: nowrap;
+}
+
+.app-c-action-link__link {
+  @include govuk-font(19, $weight: bold);
+
```

Comment:
probably want to remove this extra line

Reply:
The blank line between 36 and 38? I think our convention is to have an empty line like this between css rules and nested class declarations.